segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/DebugBookKeeperLogWrapper.java

```diff
@@ -264,6 +272,19 @@ public void deleteDurableLogMetadata() throws DurableDataLogException {
         }
     }
 
+    public void deleteLedgersStartingWithId(long startId) throws DurableDataLogException {
```

Comment:
Question: is it possible to do the actual deletion in this method without needed to create another method with the same name in `BookkeeperLog`? I mean, is it possible to invoke here `Ledgers.delete()`?

Reply:
updated as suggested